src/os/bluestore/BlueStore.cc

```diff
@@ -3569,7 +3569,8 @@ bid_t BlueStore::ExtentMap::allocate_spanning_blob_id()
 
 void BlueStore::ExtentMap::reshard(
   KeyValueDB *db,
-  KeyValueDB::Transaction t)
+  KeyValueDB::Transaction t,
+  uint32_t segment_size)
```

Comment:
please log segment_size in the first dout below

Reply:
Done.

Reply:
A question (as I don't know enough about rocksdb internals): is 'unit32_t' large enough?